cime/config/e3sm/config_grids.xml

```diff
@@ -2842,6 +2872,23 @@
       <map name="LND2ATM_SMAPNAME">cpl/gridmaps/fv0.23x0.31/map_fv0.23x0.31_to_ne120np4_aave_110331.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne120np4" lnd_grid="r0125">
+      <map name="ATM2LND_FMAPNAME">lnd/clm2/mappingdata/maps/ne120np4/map_ne120np4_to_0.125_nomask_aave.160613.nc</map>
+      <map name="ATM2LND_SMAPNAME">lnd/clm2/mappingdata/maps/ne120np4/map_ne120np4_to_0.125_nomask_aave.160613.nc</map>
```

Comment:
You are using mono maps for ne30<->r0125, but aave maps for ne120<->r0125? Should we use mono for both? Pinging @mt5555

Reply:
Yes, we should.  For np4 grids, mono is slightly better than aave.   Probably not worth changing, since this is a v1 compset (E3SM v2 atmosphere will be using "pg2" grid, and thus not this compset).  

Based on this page:

https://acme-climate.atlassian.net/wiki/spaces/ED/pages/178848194/Transition+to+TempestRemap+for+Atmosphere+grids 

ATM2LND_FMAPNAME is an SE->FV flux map, so my recommendation is "mono"
ATM2LND_SMAPNAME is an SE->FV state map, so my recommendation is "intbilin"

Reply:
Changed to use mono maps. intbilin map generated using ncremap with mwf method failed to pass check. May revisit later.